### PR TITLE
added tinyMCE rte and configured babel

### PR DESCRIPTION
**Pull request by jeremystephencobb**
Ok there's a fair amount going on here!

- Babel works with an ES2015 configuration, see ya `consts` 🎉 

- I addded tinyMCE, our shiny new rich text editor. Basically adding this as a node module is only half the battle, as you need the styles to get it to work. I added a the styles to our `/stylesheets` and added some file and font loaders to the `webpack.config`in order to compile the fonts and stuff down.

- The tinyMCE code needs to initialize on a DOM element, which proved very tricky. For now it's working on a `setTimeout...` but I feel like this is a poor solution.
  - I _think_ we might need to look into a sort of phony DOM solution into the Node app, but I'm not sure.

- the RTE is initialized on a textarea just like with the basic form, so the posts save properly

### CODE REVIEW

**Comment by jeremystephencobb** [comment]
Lol I tried to add the `/public/dist` to the gitignore but couldn't get it 😢

**Comment by jeremystephencobb** [comment]
- ~ The tinyMCE code needs to initialize on a DOM element, which proved very tricky. For now it's working on a `setTimeout...` but I feel like this is a poor solution.
  - I _think_ we might need to look into a sort of phony DOM solution into the Node app, but I'm not sure.~

^^ nevermind, I added a `defer` tag to the script in `layout.pug` and that sorted it

**Comment by andrewmcgov** [comment]
#4

File: webpack.config.js [hunk 1]
```diff
@@ -17,7 +17,63 @@ module.exports = {
...
+          use: ["css-loader"],
+          fallback: "style-loader"
+        })
+      },
+      {
+        test: /\.js$/,
+        exclude: /(node_modules|bower_components)/,
+        use: [
+          {
+            loader: "babel-loader",
```
Comment (andrewmcgov) [inline webpack.config.js]: Lets go over this at some point, I really want to understand every part of this file haha.

**Pull request merged**